lib/driver.js

```diff
@@ -332,6 +324,14 @@ class IosDriver extends BaseDriver {
       return;
     }
 
+      // if we have an ipa file, set it in opts
+      if (this.opts.app) {
```

Comment:
oh. I guess we can install .apps as well as .ipas.

Still, something seems off architecturally here. I'm fine with your fix but I think it highlights a pre-existing bit of conceptual debt. @imurchie what do you think?